### DISCUSSION TITLE
Thieves guild window improvement for "zero" hero

### DISCUSSION THREAD

**Discussion by Branikolog** [ihhub/fheroes2#6706] [Ideas]
Currently, there's an empty space in information window when any player has no heroes.
![image](https://user-images.githubusercontent.com/55348946/221164390-73e5ab3e-b847-4335-92d2-5cf67c6151cc.png)

I propose adding an empty frame where the strongest hero portrait is supposed to be placed:
![Tguild](https://user-images.githubusercontent.com/55348946/221164530-f1b5cbee-ff2d-4adc-a6d8-dd830b68ba0f.png)

**Comment by Districh-ru** [comment]
Hello @Branikolog, I think this proposal might help some players understand for sure that the Thieves Guild is providing information on the best hero and the enemy has no hero. Especially in the shown case of two players, some may think that the current number of Thieves Guilds simply does not provide information about the best enemy hero. :)

**Comment by LeHerosInconnu** [comment]
Hello @Branikolog,

An empty frame looks too... empty. :D
Maybe a reduced version of the player's blazon, or at least a graphic with the player's color would look more aesthetic.

**Reply by Mr-Bajs** [reply]
It looks too empty. Why not replace the empty frame with an framed picture of the random race that you select when picking a race/faction?

**Reply by Branikolog** [reply]
Well, emptiness is exactly the thing is happening in enemy kingdom in this particular case. There're no heroes in enemy kingdom. :) So the frame is empty. Absolutely demonstrative interpretation of zero enemy heroes. :)
 
> It looks too empty. Why not replace the empty frame with an framed picture of the random race that you select when picking a race/faction?

That icon is too large.

![Empty](https://user-images.githubusercontent.com/55348946/226106451-285598db-613e-43aa-9d83-3fb9ddc0b060.png)

Isn't this UI element seem empty? Zero hero frame is supposed to show the lack of hero. :D
